src/main/java/frc/robot/commands/vision/VisionAim.java

```diff
@@ -20,7 +21,7 @@
   private Robot robot;
 
   // CONSTANTS //
-  final static double TURN_SPEED = 0.4;
+  static final double TURN_SPEED = 0.4;
```

Comment:
Should this be removed if it is no longer used? Replaced by `Drivetrain.AIM_TURN_SPEED`

Reply:
Yes, that should be removed. Thanks.